site/components/pages/Buy/CreditCardTimeline/index.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { FC, ReactNode } from "react";
+import { Anchor as A, Text } from "@klimadao/lib/components";
+
+import * as styles from "./styles";
+import { Trans } from "@lingui/macro";
+import { urls } from "@klimadao/lib/constants";
+
+const creditCardContent: CardContent[] = [
+  {
+    title: <Trans id="buy.beginner_01.title">Install Metamask</Trans>,
+    description: (
+      <div className={styles.description}>
+        <Text color="lighter">
+          <Trans id="buy.beginner_01.description_01">
+            <A href="https://metamask.io/">Metamask</A> is the most popular
+            wallet for Ethereum and Polygon. This is what you use to connect to
+            our app and send transactions. Be sure to install the extension for
+            your web browser.
+          </Trans>
+        </Text>
+      </div>
+    ),
+  },
+  {
+    title: <Trans id="buy.beginner_02.title">Connect to our app</Trans>,
+    description: (
+      <div className={styles.description}>
+        <Text color="lighter">
+          <Trans id="buy.beginner_02.description_01">
+            With the Metamask browser extension installed and your new wallet
+            set up, head to{" "}
+            <A href={urls.buy_dapp}>app.klimadao.finance/#/buy</A> and click
```

Comment:
urls.buy_dapp still points to dapp.klimadao.finance - should be updated to app.klimadao.finance